kubernetes-services/work-updater/README.md

```diff
@@ -0,0 +1,35 @@
+# Harmony Work Updater
+
+This folder contains the code and build scripts for the Harmony work updater,
+which is responsible for reading work-item updates from a queue and handling processing them.
+
+## Building the Work Updater
+
+1. Run `npm build` in this directory to build with the tag `latest`. To use a different tag,
```

Comment:
Is this correct? Since there are two Dockerfiles it seems like we might have to run something different for different OS'es.

Reply:
This will always build for the local environment. There is a section below about building for and deploying to the sandbox.